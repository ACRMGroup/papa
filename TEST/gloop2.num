L1 D
L2 I
L3 Q
L4 M
L5 T
L6 Q
L7 S
L8 P
L9 S
L10 S
L11 L
L12 S
L13 A
L14 S
L15 L
L16 G
L17 E
L18 R
L19 V
L20 S
L21 L
L22 T
L23 C
L24 R
L25 A
L26 S
L27 Q
L28 E
L29 I
L30 S
L31 G
L32 Y
L33 L
L34 S
L35 W
L36 L
L37 Q
L38 Q
L39 K
L40 P
L41 D
L42 G
L43 T
L44 I
L45 K
L46 R
L47 L
L48 I
L49 Y
L50 A
L51 A
L52 S
L53 T
L54 L
L55 D
L56 S
L57 G
L58 V
L59 P
L60 K
L61 R
L62 F
L63 S
L64 G
L65 R
L66 R
L67 S
L68 G
L69 S
L70 D
L71 Y
L72 S
L73 L
L74 T
L75 I
L76 S
L77 S
L78 L
L79 E
L80 S
L81 E
L82 D
L83 F
L84 A
L85 D
L86 Y
L87 Y
L88 C
L89 L
L90 Q
L91 Y
L92 L
L93 S
L94 Y
L95 P
L96 L
L97 T
L98 F
L99 G
L100 A
L101 G
L102 T
L103 K
L104 L
L105 E
H1 Q
H2 V
H3 Q
H4 L
H5 Q
H6 Q
H7 S
H8 G
H9 T
H10 E
H11 L
H12 A
H13 R
H14 P
H15 G
H16 A
H17 S
H18 V
H19 R
H20 L
H21 S
H22 C
H23 K
H24 A
H25 S
H26 G
H27 Y
H28 T
H29 F
H30 T
H31 T
H32 F
H33 G
H34 I
H35 T
H36 W
H37 V
H38 K
H39 Q
H40 R
H41 T
H42 G
H43 Q
H44 G
H45 L
H46 E
H47 W
H48 I
H49 G
H50 E
H51 I
H52 F
H52A P
H53 G
H54 N
H55 S
H56 K
H57 T
H58 Y
H59 Y
H60 A
H61 E
H62 R
H63 F
H64 K
H65 G
H66 K
H67 A
H68 T
H69 L
H70 T
H71 A
H72 D
H73 K
H74 S
H75 S
H76 T
H77 T
H78 A
H79 Y
H80 M
H81 G
H82 L
H82A S
H82B S
H82C L
H83 T
H84 S
H85 E
H86 D
H87 S
H88 A
H89 V
H90 Y
H91 F
H92 C
H93 A
H94 R
H95 E
H96 I
H101 R
H102 Y
H103 W
H104 G
H105 Q
H106 G
H107 T
H108 T
H109 L
H110 T
H111 V
