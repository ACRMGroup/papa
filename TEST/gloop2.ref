Predicted packing angle: -47.619
