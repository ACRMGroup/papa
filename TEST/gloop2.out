Predicted packing angle: 47.6190
